Doc/reference/compound_stmts.rst

```diff
@@ -325,6 +330,47 @@ when leaving an exception handler::
    >>> print(sys.exc_info())
    (None, None, None)
 
+.. index::
+   keyword: except_star
+
+The :keyword:`except*<except_star>` clause(s) are used for handling
+:exc:`ExceptionGroup`s. The exception type for matching is interpreted as in
+the case of :keyword:`except`, but in the case of exception groups we can have
+partial matches when the type matches some of the exceptions in the group.
+This means that multiple except* clauses can execute, each handling part of
+the exception group. Each clause executes once and handles an exception group
+of all matching exceptions.  Each exception in the group is handled by at most
+one except* clause, the first that matches it. ::
```

Comment:
Do these need markup?
```suggestion
This means that multiple ``except*`` clauses can execute, each handling part of
the exception group. Each clause executes once and handles an exception group
of all matching exceptions.  Each exception in the group is handled by at most
one ``except*`` clause, the first that matches it. ::
```

Reply:
Good question. In the rest of the doc I saw that except is marked up as a keyword a couple of times and most of the time not marked up at all.  So I did the same for except* but open to changing it.

Reply:
Alright. If the rest of the `try-except` section doesn't use markup, I think it's good to keep it as-is for consistency. (some other parts of the document use markup for things like ``else``, but IMO consistency within a single section is more important)